### DISCUSSION TITLE
NGAlerts with AWS Autoscaling

### DISCUSSION THREAD

**Discussion by Darth-Bobo** [grafana/grafana#37974] [Alerting]
I was about to try out the new alerting system in our test environment, but I see that the Opt-In instructions says it, "stores the notification log and silences on disk".

I have two questions:
1. Does that mean I will see alerts fired for every node in the cluster when they trigger?
2. Will duplicate/old alerts fire when a new node joins the cluster?

I can't find any documents that mention the new alerts and clustering, but if anyone can point me to them then that would be a great help.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
